core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/reconfigure-rabbitmq-app.yml

```diff
@@ -7,8 +7,9 @@
   block:
     - name: after-patching | Get rabbitmq namespace
       shell: |-
-        kubectl get statefulsets.apps --all-namespaces -o=jsonpath='{range .items[*]}{.metadata.namespace}{"\t"}{.spec.template.spec.containers[].image}{"\n"}{end}'|
-        awk '/rabbitmq/ {print $1}'
+        kubectl get statefulsets.apps --all-namespaces \
```

Comment:
@atsikham how about replacing all that ugly code with something like this (:thinking:):
```yaml
- command: kubectl get statefulsets --all-namespaces --output json
  register: shell_get_statefulsets
  changed_when: false

- command: kubectl delete pods --namespace {{ item }} --all
  loop: "{{ _namespaces }}"

  when: _namespaces | length > 0

  vars:
    _namespaces: >-
      {%- set output = [] -%}
      {%- for item in (shell_get_statefulsets.stdout | from_json)["items"] -%}
        {%- for container in item.spec.template.spec.containers -%}
          {%- if '/rabbitmq:' in container.image -%}
            {{- output.append(item.metadata.namespace) -}}
          {%- endif -%}
        {%- endfor -%}
      {%- endfor -%}
      {{- output -}}
```

Reply:
@sk4zuzu I suggest to create another PR for that as testing scope is different. I changed that line and it's showed here as there was a linter warning.

Reply:
That's fine! If it gets there somehow I'm :+1::heart_eyes: